sdk/python/packages/flet-core/src/flet_core/drag_target.py

```diff
@@ -149,11 +147,12 @@ def _get_control_name(self):
         return "dragtarget"
 
     def _get_children(self):
-        children = []
-        if self.__content:
-            self.__content._set_attr_internal("n", "content")
-            children.append(self.__content)
-        return children
+        self.__content._set_attr_internal("n", "content")
+        return [self.__content]
+
+    def did_mount(self):
+        super().did_mount()
+        assert self.__content.visible, "content must be visible"
 
```

Comment:
Yeah, `did_mount` is not the right place to do the checks. Let's fix it everywhere?